cmd/utils/flags.go

```diff
@@ -128,7 +128,7 @@ var (
 	}
 	NetworkIdFlag = cli.Uint64Flag{
 		Name:  "networkid",
-		Usage: "Network identifier (integer, 1=Frontier, 3=Ropsten, 4=Rinkeby, 5=Görli)",
+		Usage: "Network identifier (integer, 1=Frontier, 3=Ropsten, 4=Rinkeby, 5=Görli), (for testnets: use --goerli, --ropsten, --rinkeby instead)",
```

Comment:
Perhaps can be made shorter? 
```suggestion
		Usage: "Explicitly set network id (integer)(For testnets: use --goerli, --ropsten, --rinkeby instead)",
```